src/lib/styles/global/modal.scss

```diff
@@ -14,21 +14,8 @@ div.modal {
   }
 
   .toolbar {
-    display: flex;
```

Comment:
Does this mean that existing usages of `.toolbar` will break unless they also add `.toolbar-buttons`?

Reply:
Yes, the plan was to add the `.toolbar-buttons` class together with the `gix-upgrade` pr.

Reply:
What do you think about moving the `toolbar` class outside of `div.modal` so we can reuse it without having to add multiple classes each time?
Or would that not work?

Reply:
Not sure, 
1. it has an extra padding inside the modal;
2. it's a bit confusing that there will be a global class `.toolbar` that is not related directly to the `Toolbar` component.

What do you think about renaming `.toolbar-buttons` in this pr to `.full-width-mobile-buttons`?

Reply:
I'm concerned that using `.toolbar` inside `div.modal` used to work and now requires an extra class.

How about also adding `.toolbar` style that works inside `BottomSheet`?

I'm also still confused how this even works. Why does the `div.modal` selector here work without the need for `:global`?

Reply:
Then why did it have `:global(button) {` on line 22?

Reply:
Good question, but it doesn't work w/o `:global`

Reply:
Thanks for checking.

Are you still looking into this:
> I'm concerned that using .toolbar inside div.modal used to work and now requires an extra class.
> How about also adding .toolbar style that works inside BottomSheet?

Or did you miss that?

Reply:
A [mixin](https://github.com/dfinity/gix-components/pull/375/files#diff-bbba86cbcfd780d497c1fc89e609c3599beee6594f8bdb01bcc8195210e97565R20) was added.
- no changes needed for modals
- can be used to fix the "Participate" button issue -- https://github.com/dfinity/nns-dapp/pull/4401

Reply:
@dskloetd, pr was updated:
- `*` selectors to make it most flexible
- keep only `base` version of the mixin